### ISSUE TITLE
China censorship copypasta and variants 

### ISSUE THREAD

**Issue by mm12** [TheCaptainYaya/Vortex#1]
thinks like
tank man 
six four 
6 4 
1989 
Tiananmen Square 
animal farm  
Winnie-the-Pooh 
陆肆 
六四 
自由花 
终身 
不要脸 
不同意

**Comment by BCDeshiG** [comment]
As much as we want to keep copypasta out of our server, let's not be welcoming the Chinese overlords in lol.

**Comment by HJBit** [comment]
isn't this censorship, so we'd be the Chinese government?

**Comment by HJBit** [comment]
plus it'd piss off the history nerds if they tried to bring it up in a meme in #history-memes and they learned it was censored

**Comment by mm12** [comment]
yeah, i'm saying to combat people who are trying to ban any user thats somehow in china

**Comment by HJBit** [comment]
lmao that's funny, rip users in china

**Comment by mm12** [comment]
Today (well, yesterday in china) was "Chinese internet maintenance day" aka the anniversary of the Tiananmen Square massacre that happened on 6/4/1989.
